StyleGuide.md

```diff
@@ -757,21 +757,19 @@ Examples help show the feature in action. Examples follow a basic format:
 
 Version tags inform users of new and updated features in CockroachDB, and could motivate users to upgrade to the latest major or minor version of CockroachDB. Version tags also help us identify new and updated features that we can call out in [our GA release notes](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/402718726/GA+Release+Checklist).
 
-To add a version tag, use the following HTML `span`:
+To add a version tag, use the following Liquid tag:
 
 ~~~
-<span class="version-tag">New in vX.X:</span>
+{% include_cached new-in.html version="v22.1" %}
 ~~~
 
 Put version tags at the beginning of a paragraph, sentence, or description in a table cell.
 
-Note: Do not use version tags in bulleted lists items. To denote a new feature in a bulleted list, start the bulleted item with "**New in vX.X:**".
-
 If a feature is new in a GA release, use the major release number for the release version tag (e.g., `New in v22.1`).
```

Comment:
I think these examples in parens are confusing. Could you change them to use the Liquid version?

Reply:
Done.